apps/localplanning.services/src/components/applications/ApplicationCard.tsx

```diff
@@ -3,84 +3,22 @@ import type { Application } from "./hooks/useFetchApplications";
 import { useDeleteApplication } from "./hooks/useDeleteApplication";
 import { formatDate } from "@lib/date";
 import { $applicationId } from "@stores/applicationId";
+import { StatusBadge } from "./StatusBadge";
 
 const ProgressText: React.FC<Application> = (application) => {
-  const progressText = (() => {
+  const date = (() => {
     switch (application.status) {
       case "draft":
-        return (
-          <>
-            Draft started
-            <strong className="font-semibold"> {formatDate(application.createdAt)}</strong>
-          </>
-        );
+        return formatDate(application.createdAt);
       case "awaitingPayment":
-        return (
-          <>
-            Application completed
-            <strong className="font-semibold"> {formatDate(application.updatedAt)}</strong>, awaiting payment
-          </>
-        );
-      case "submitted":
-        return (
-          <>
-            Application sent
-            <strong className="font-semibold"> {formatDate(application.submittedAt!)}</strong>
-          </>
-        );
-    }
-  })();
-
-  return (
-    <span className="text-body-md mb-0">
-      {progressText}
-    </span>
-  )
-};
-
-// TODO: How should we handle applications without progress indicators?
-const ProgressBar: React.FC<Application> = (application) => {
-  const progressColour = (() => {
-    switch (application.status) {
+        return formatDate(application.updatedAt);
       case "submitted":
-        return "bg-green-600";
-      case "awaitingPayment":
-        return "bg-red-900";
-      default:
-        return "bg-black";
+        return formatDate(application.submittedAt!);
```

Comment:
```suggestion
        return formatDate(application.submittedAt);
```

Type narrowing via the `switch/case` should make this a `SubmittedApplication` so the `!` shouldn't be needed.

Were you hitting type issues here?